types/unist/unist-tests.ts

```diff
@@ -1,56 +1,93 @@
-import { Data, Point, Position, Node, Literal, Parent } from 'unist';
+import { Point, Position, Node, Literal, Parent, NodeData } from 'unist';
 
-const data: Data = {
+const data = {
     string: 'string',
     number: 1,
     object: {
-        key: 'value'
+        key: 'value',
     },
     array: [],
     boolean: true,
-    null: null
+    null: null,
```

Comment:
These comma changes make it a bit hard to review this PR

Reply:
It is changed by prettier.

Reply:
It’s generally appreciated to not change style in the same PR as where you change code.

I am guessing that you have a different Prettier setting than this repo?

Reply:
No, it is run by `yarn prettier --write`